test/retorts.py

```diff
@@ -100,14 +100,32 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self.request_mock.passthru_prefixes = self.passthru_prefixes
 
 
-class AuthResponseHelper(ResponsesHelper):
+class AuthResponseHelper:
 
     def __init__(self, passthru_url: str = None, request_mock: responses.RequestsMock = None) -> None:
-        super().__init__(request_mock)
+        # noinspection PyProtectedMember
+        self.request_mock = responses._default_mock if request_mock is None else request_mock
```

Comment:
This looks like it does everything that ResponsesHelper did. If we can eliminate the workarounds that ResponsesHelper implements from many tests why do we have to maintain them here. It looks like your'e also removing the doctest while maintaining the code that the doctest covered. That's a reduction in coverage, if I am not mistaken.

Reply:
AuthResponseHelper  was removed w/ #2969